webapp/platform/types/src/properties.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export type PropertyField = {
+    id: string;
+    group_id?: string;
```

Comment:
nit: this is mandatory

```suggestion
    group_id: string;
```